js/index.js

```diff
@@ -0,0 +1,98 @@
+import lozad from "lozad";
+import checkElement from "../utils/dom";
+
+import Menu from "./models/menu";
+import * as menuView from "./views/menuView";
+
+import * as navMenus from "../utils/navMenus";
+
+const state = {};
+
+function addDisplayOnHover() {
+  const elementContained = document.querySelectorAll(".article");
+  let clickedElement;
+  Array.from(elementContained).forEach(async elem => {
+    elem.addEventListener("mouseover", function() {
+      const id = elem.getAttribute("class").split(" ");
```

Comment:
Its a little confusing get the attribute from a `class` and call it `id` 😅

Reply:
help me with that, how i should call it, give me examples 🙏.